content/en/container-orchestration.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Container Orchestration
+status: Draft
+category: Concept
+---
+
+## What it is
+[Container](/container/) orchestration provides the ability to manage and automate the lifecycle of containerized applications in dynamic environments. 
+This includes different aspects that are also pursued by [Cloud Native Technologies](/cloud-native-tech/) and [Cloud Native Apps](/cloud-native-apps/), such as: deployment, (auto)scaling, auto healing and monitoring.
+It works like a conductor of an orchestra, who makes sure that every part of the orchestra does what it should.
+
+## Problem it addresses 
+The rise of [Cloud Native Apps](/cloud-native-apps/) and [technologies](/cloud-native-tech/) as well as the fast-changing demands of modern businesses introduced a new set of problems. 
+Managing [microservices](/microservices), security, and network-related things at scale, as well as [distributed systems](/distributed-systems) in general, are hard if not impossible to manage manually.   
+
+## How it helps
+Container orchestration provides a declarative method of automating much of the work, which makes the operational complexity manageable for software engineering teams.
+This includes a variety of tasks, such as provisioning, deployment, scaling (both up and down), networking, load balancing, and other activities.
```

Comment:
```suggestion
This automation simplifies many of the engineering teams' otherwise highly manual and complex operational tasks, including provisioning, deployment, scaling (up and down), networking, load balancing, and other activities.
```